### DISCUSSION TITLE
下拉刷新

### DISCUSSION THREAD

**Discussion by lrx1994** [openkraken/kraken#721] [General]
北海目前有支持下拉刷新和上拉加载嘛,具体是如何实现呢,有无参考文献,求大佬告知

**Comment by wssgcg1213** [comment]
没有支持，北海的定位更类似 Chromium, 下拉和上拉的行为建议在外部的 Shell 上实现; 内部的手势事件传递有现成的能力可以对接, 可以参考 Gesture

**Comment by pba-cra** [comment]
![refresh](https://user-images.githubusercontent.com/14161857/140469035-1f721ec2-21a9-4efe-957d-8749796d2bd9.gif)
我们这边已经实现了这个下拉刷新的效果，纯前端实现的阻尼。修改了kraken的scroller的scrollPhysic.不然两个阻尼的效果冲突。
中端andorid机器 profile 模式上可以跑到50-55帧
